docs/game_sdk/Applications.md

```diff
@@ -105,6 +114,41 @@ applicationManager.ValidateOrExit((result) =>
 });
 ```
 
+## GetTicket
+
+Get the encrypted app ticket for the current user. This is a base64 encoded string which contains:
+
+- the application id tied to the ticket
+- the user's user id
+- a timestamp for the ticket
+- the list of the user's [entitlements](#DOCS_GAME_SDK_STORE/data-models-entitlement-struct) for the application id
+
+These values can be accessed by decrypting the ticket with your application's private key and transforming the string into a [DecryptedAppTicket](#DOCS_GAME_SDK_APPLICATIONS/data-models-decryptedappticket-struct). The ticket is encrypted using [libsodium](https://github.com/jedisct1/libsodium) which should be available for any programming language.
```

Comment:
We are using signing, not encryption.

- https://download.libsodium.org/doc/public-key_cryptography/public-key_signatures

Bindings for other languages include. 

- https://download.libsodium.org/doc/bindings_for_other_languages 

The structure of a ticket is

<version>.<signature>.<base64 json data>

So you want to split by dot. for now check version is 2.
Get the public key from us directly for now (later in dev portal)

Using this C# example here

https://bitbeans.gitbooks.io/libsodium-net/content/public-key_cryptography/public-key_signatures.html

We want detached mode.

You pass the signature, the message (aka base64/json) and the public key

If it returns true then you can base64 decode and json deserialize :)